openhands/runtime/README.md

```diff
@@ -23,9 +23,9 @@ Key features of the `Runtime` class:
 - Action execution methods for different types of actions (run, read, write, browse, etc.)
 - Abstract methods for file operations (to be implemented by subclasses)
 
-### 2. client/client.py
+### 2. server.py
 
-The `client.py` file contains the `RuntimeClient` class, which is responsible for executing actions received from the OpenHands backend and producing observations. This client runs inside a Docker sandbox.
+The `server.py` file contains the `RuntimeClient` class, which is responsible for executing actions received from the OpenHands backend and producing observations. This client runs inside a Docker sandbox.
```

Comment:
This is a bit confusing, because it's mixing "client" and "server". What about naming it something more descriptive (both in the file name and the class name), such as `ActionRunner` or `ActionExecutor`?

Reply:
Maybe we can rename `RuntimeClient` to `ActionExecutor` and `server.py` to `action_execution_server.py`?

Reply:
done in most recent commit!